superset/assets/src/visualizations/nvd3/utils.js

```diff
@@ -37,6 +37,20 @@ export function drawBarValues(svg, data, stacked, axisFormat) {
     });
 }
 
+// Formats the series key to account for a possible NULL value
+function getFormattedKey(seriesKey, toDompurify) {
+    let key = '';
```

Comment:
can you make this more concise with a ternary operator? 

`const key = shouldDomPurify ? dompurify.sanitize(seriesKey) : seriesKey`